scripts/joysetup.sh

```diff
@@ -423,38 +462,38 @@ function create_zpool
 
     if [[ -n ${MOCKCN} ]]; then
         # setup initial usage
-        json -e "this.usage = $(($RANDOM * $RANDOM))" \
-            < ${POOL_JSON} > ${POOL_JSON}.new \
-            && mv ${POOL_JSON}.new ${POOL_JSON}
+        json -e "this.usage = $(( RANDOM * RANDOM))" \
+            < "${POOL_JSON}" > "${POOL_JSON}.new" \
+            && mv "${POOL_JSON}.new" "${POOL_JSON}"
         echo "Not checking NTP in mock CN."
         return;
     fi
 
     [[ -n "$ENCRYPTION" ]] && e_flag="-e "
```

Comment:
Considering that we merged encryption from ZoL, in theory it could, but at this point I'm considering it not supported. There may be some specific triton-ish things we need to work out before it's actually viable.